lib/atlas/dataset/derived_dataset.rb

```diff
@@ -20,14 +20,19 @@ def graph_path
 
     # Overwrite
     def dataset_dir
-      @dataset_dir ||=
-        Atlas.data_dir.
-          join(DIRECTORY).
-          join(Dataset::FullDataset.find(base_dataset).key.to_s)
+      @dataset_dir ||= Atlas.data_dir.join(DIRECTORY, full_dataset.key.to_s)
+    end
+
+    def scaling_factor
+      number_of_residences / full_dataset.number_of_residences
```

Comment:
We can just remove this because the `scaling_factor` method already exists in the `Scaler`.